spec/support/koala_test.rb

```diff
@@ -241,8 +241,8 @@ def self.activate_adapter!
           require 'typhoeus/adapters/faraday' if adapter.to_s == "typhoeus"
           Faraday.default_adapter = adapter.to_sym
         end
-      rescue ParserError
-        puts "Unable to load adapter #{adapter}, using Net::HTTP."
+      rescue => e
+        puts "Unable to load adapter #{adapter}, using Net::HTTP. #{e.class} #{e.message}"
```

Comment:
LIVE test can't be run with typhoeus because there is no adapter for faraday 2 yet. Maybe one day cf: https://github.com/typhoeus/typhoeus/issues/686